hasher-matcher-actioner/hmalib/scripts/migrations/2022_04_02_default_content_signal_type_configs.py

```diff
@@ -13,40 +14,62 @@
     ToggleableSignalTypeConfig,
     full_class_name,
 )
-
+from hmalib.common.logging import get_logger
 from hmalib.scripts.migrations.migrations_base import MigrationBase
 
+logger = get_logger(__name__)
+
 
 class _Migration(MigrationBase):
     def do_migrate(self):
-        create_config(
-            ToggleableContentTypeConfig(
-                name=ToggleableContentTypeConfig.get_name_from_type(VideoContent),
-                content_type_class=full_class_name(VideoContent),
-                enabled=True,
+        """
+        if any of the 4 default mappings do not exist, create them
+        """
+        try:
+            create_config(
+                ToggleableContentTypeConfig(
+                    name=ToggleableContentTypeConfig.get_name_from_type(VideoContent),
+                    content_type_class=full_class_name(VideoContent),
+                    enabled=True,
+                )
             )
-        )
-
-        create_config(
-            ToggleableContentTypeConfig(
-                name=ToggleableContentTypeConfig.get_name_from_type(PhotoContent),
-                content_type_class=full_class_name(PhotoContent),
-                enabled=True,
+        except ClientError:
+            logger.warning(
+                "Attempted to add ToggleableContentTypeConfig for VideoContent, but it already exists."
```

Comment:
blocking: We do not check the error code of this to see if this is actually the case, and so could accidentally eat some serious error. Is there something in the exception we can get?

Reply:
I believe only kind of ClientError we can see is this one, I believe everything else should be different as it would be an AccessError or Network error instead. This is matching the existing checks used elsewhere in this flow: https://github.com/facebook/ThreatExchange/blob/main/hasher-matcher-actioner/hmalib/common/configs/tx_apis.py#L81

Reply:
The existence of other checks with this same problem is not reassuring to me!

Where do ClientErrors come from? Can we add codes to them?